<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnThoat">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Thoát</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblNhanVien">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>390</y>
      <width>501</width>
      <height>211</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>IDNV</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Họ Tên</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tên đăng nhập</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>mật khẩu</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>VaiTrò</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="txtIDNV">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>ID Nhân Viên</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTenDangNhap">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Tên đăng nhập</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMatKhau">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Mât khẩu</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtHoTen">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Họ Tên</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbVaiTro">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>50</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Admin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NhanVien</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnThem">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Thêm </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSua">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Sửa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnXoa">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>230</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Xóa</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTuKhoa">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Tìm Kiếm Nhân Viên</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTimKiem">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>310</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Tìm Kiếm</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
